src/docfx/Errors.cs

```diff
@@ -392,6 +392,12 @@ public static Error AtXrefNotFound(SourceInfo<string> source)
             public static Error XrefNotFound(SourceInfo<string> source)
                 => new Error(ErrorLevel.Warning, "xref-not-found", $"Cross reference not found: '{source}'.", source);
 
+            public static Error InvalidXrefType(SourceInfo<string> xref, string expectedXrefType, string realXrefType)
+               => new Error(ErrorLevel.Warning, "xref-invalid-type", $"Cross reference '{xref}' invalid type, which is expected '{expectedXrefType}' but now is '{realXrefType}'.", xref);
```

Comment:
>which is expected '{expectedXrefType}' but now is '{realXrefType}' [](start = 110, length = 66)

'{expectedXrefType}' is expected but now is '{realXrefType}'

Reply:
OK